plover/gui_qt/dictionaries_widget.py

```diff
@@ -288,14 +332,25 @@ def _set_selection(self, row_list):
     def on_selection_changed(self):
         if self._updating:
             return
-        enabled = bool(self.table.selectedItems())
-        for action in (
+        selection = self._get_selection()
+        has_selection = bool(selection)
+        for widget in (
             self.action_RemoveDictionaries,
-            self.action_EditDictionaries,
             self.action_MoveDictionariesUp,
             self.action_MoveDictionariesDown,
         ):
-            action.setEnabled(enabled)
+            widget.setEnabled(has_selection)
+        has_live_selection = next((
+            True for row in selection
+            if self._config_dictionaries[row].path
+            in self._loaded_dictionaries
+        ), False)
```

Comment:
Isn't this just `any(self._config_dictionaries[row].path in self._loaded_dictionaries for row in selection)`?